client/src/components/Disclaimer.js

```diff
@@ -1,29 +1,45 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
 import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
 import Button from 'react-bootstrap/Button';
 
 class Disclaimer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showPopUp: true,
+    };
```

Comment:
I feel like a better place to handle this logic would be in `DiagnosticContainer` using Redux.

We have an initial state of `true` for `showPopUp` in Redux. Then, when rendering `DiagnosticContainer`, we determine if we need to render `Disclaimer` at all.

Reply:
gotcha. Should I do that for this PR or make it a separate thing?